### PR TITLE
common: add writable request streams

### PR DESCRIPTION
**Pull request by stephenplusplus**
This is pulled out from the Speech PR (#1407) to add a writable request stream.

### CODE REVIEW

File: packages/common/src/grpc-service-object.js [hunk 1]
```diff
@@ -109,17 +109,26 @@ GrpcServiceObject.prototype.setMetadata = function(metadata, callback) {
  *
  * @private
  */
-GrpcServiceObject.prototype.request = function(protoOpts, reqOpts, callback) {
-  return this.parent.request(protoOpts, reqOpts, callback);
+GrpcServiceObject.prototype.request = function() {
+  return this.parent.request.apply(this, arguments);
```
Comment (callmehiphop) [inline packages/common/src/grpc-service-object.js]: Should `this.parent` be the context here?

Comment (stephenplusplus) [inline packages/common/src/grpc-service-object.js]: Oops.

Comment (stephenplusplus) [inline packages/common/src/grpc-service-object.js]: Fixed.

File: packages/common/src/grpc-service.js [hunk 2]
```diff
@@ -293,7 +294,7 @@ GrpcService.prototype.request = function(protoOpts, reqOpts, callback) {
  * Make an authenticated streaming request with gRPC.
  *
  * @param {object} protoOpts - The proto options.
- * @param {string} protoOpts.service - The service git stat.
```
Comment (callmehiphop) [inline packages/common/src/grpc-service.js]: lol

**Pull request merged**